lib/plugins/package/lib/zipService.js

```diff
@@ -2,12 +2,15 @@
 
 /* eslint-disable no-use-before-define */
 /* eslint-disable no-param-reassign */
+/* eslint-disable consistent-return */
```

Comment:
Which lines are affected by this? Can the implementation be changed to comply to the rule, or is there a specific reason, why this cannot be done?